ckan/cli/cli.py

```diff
@@ -31,19 +32,33 @@
 log = logging.getLogger(__name__)
 
 
+class IClick(click.Group):
```

Comment:
I think, everyone who looks over the code will assume that `IClick` is some kind of interface(i.e. something that acts like contract and requires future implementation), whereas in your case it's Group that initializes the application and makes re-attempt to invoke the missing command